### PR TITLE
An operation to create a spreadsheet from profiles

### PR DESCRIPTION
**Pull request by bryantaustin13**
Added an operation to create a spreadsheet with ValueSet from structure definitions within a profile.

**Description**
Added operation "-ProfilesToSpreadsheet" to create a spreadsheet from profile definitions.json showing the StructureDefinitions and the element binding valuesets.
When the tests are run the spreadsheet should be produced: target/test-output/profileToSpreadsheet/QI Core Valueset Analysis5.xlsx.
Addresses ESAC Jira ticket CQF-1175 and CQF-1168.

- Github Issue:  
- [ X] I've read the contribution guidelines 
- [ X] Code compiles without errors
- [ X] Tests are created / updated
- [ ] Documentation is created / updated

By creating this PR you acknowledge that your contribution will be licensed under Apache 2.0

### CODE REVIEW

File: src/main/java/org/opencds/cqf/tooling/acceleratorkit/StructureDefinitionBaseVisitor.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package org.opencds.cqf.tooling.acceleratorkit;
+
+import org.hl7.fhir.r4.model.ElementDefinition;
+import org.hl7.fhir.r4.model.StructureDefinition;
+
+import java.util.List;
+
+public class StructureDefinitionBaseVisitor {
+
+    public void visitContact(StructureDefinition sd) {
```
Comment (mdnazmulkarim) [inline src/main/java/org/opencds/cqf/tooling/acceleratorkit/StructureDefinitionBaseVisitor.java:10]: This class is a candidate to be abstract. Just an option to consider, no big deal!

**Comment by mdnazmulkarim** [comment]
A java-doc can be added here `/src/main/java/org/opencds/cqf/tooling/Main.java` describing the operation.

**Comment by mdnazmulkarim** [comment]
Looks good to me!

**Review by mdnazmulkarim — APPROVED** [review]

**Comment by brynrhodes** [comment]
This PR is maybe too big? Can we pare this down to a test case that validates the main functionality, like a few of the more complicated QICore profiles and only the dependencies of those profiles? It doesn't seem like a good idea to include the entire definition schemas for FHIR, USCore and QICore in the repository here.

**Comment by brynrhodes** [comment]
It's so big I can't even get github to let me leave that comment as a request changes PR...

**Pull request closed without merging**